src/components/ProductList/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const ProductList = () => {
+  return (
+    <section className="product-list">
+      <h2 className="product-list__title">Product List</h2>
+      <p className="product-list__description">This is a sample description</p>
+    </section>
```

Comment:
If you use a ``section``, I think it should contain a ``header`` tag inside.

Reply:
+1, would better to use `<header>` to group `<h>` and `<p>`.

Reply:
And since this is a block, make sure to wrap this with `div`.